.github/workflows/pythonpublish.yaml

```diff
@@ -17,6 +17,19 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install setuptools wheel twine
+      - name: Bump version and push tag
```

Comment:
The trigger for this workflow is a release creation.. I think if we want to automatically bump versions, it should be on master merge...